### PR TITLE
Document possible ranlib issue with removing the index

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

File: site/source/docs/compiling/Building-Projects.rst [hunk 1]
```diff
@@ -324,7 +324,17 @@ It is also worth looking at the build scripts in the `ammo.js <https://github.co
 Troubleshooting
 ===============
 
-- Make sure to use bitcode-aware *llvm-ar* instead of *ar* (which may discard code). *emmake* and *emconfigure* set the AR environment variable correctly, but a build system might incorrectly hardcode *ar*.
+- Make sure to use ``emar`` (which calls ``llvm-ar``), as the system ``ar`` may
+  not support LLVM bitcode files. ``emmake`` and ``emconfigure`` set the AR
```
Comment (sbc100) [inline site/source/docs/compiling/Building-Projects.rst]: Or wasm object file

Comment (kripken) [reply site/source/docs/compiling/Building-Projects.rst]: Is that an actual issue, though? I know we saw this with bitcode files, did we with wasm object files?

Comment (sbc100) [reply site/source/docs/compiling/Building-Projects.rst]: Definitely yes.    The native tools don't understand object file or bitcode file, they only understand ELF files.

Comment (kripken) [reply site/source/docs/compiling/Building-Projects.rst]: Thanks, updating.

**Pull request merged**